packages/hop-node/src/cli/shared/shared.ts

```diff
@@ -23,12 +26,15 @@ export function actionHandler (fn: Function) {
       if (configFilePath) {
```

Comment:
nitpick: if we now validate the config values outside of the `if (configFilePath)`, does that mean that we will always throw in `validateConfigValues(globalConfig)` if a path doesn't exist?

If so, then we can probably just do `if (!configFilePath) { throw new Error }` instead of this `if...else`.

Reply:
We can't throw here because this is a global handler and some commands don't require a config file (like keystore generation). I removed the `else` and throw in the hopNode command instead https://github.com/hop-protocol/hop/pull/360/commits/427827fe97b6fda91c4541ca4e50bd89dab7ee11